AmplifyPlugins/API/AWSAPICategoryPluginTests/Interceptor/SubscriptionInterceptor/IAMAuthInterceptorTests.swift

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import Amplify
+import AppSyncRealTimeClient
+import AWSCore
+@testable import AWSAPICategoryPlugin
+@testable import AmplifyTestCommon
+
+class IAMAuthInterceptorTests: XCTestCase {
```

Comment:
nit: do we also want to exercise `IAMAuthenticationHeader` APIs without any additional header?